config/locales/views/assignments/en.yml

```diff
@@ -28,8 +28,11 @@ en:
     hidden: "%{assignment_text} (hidden)"
     manage_course_work: Manage Course Work
     marking_scheme: 'Marking Scheme: %{identifier}'
+    maximum_file_size: 'Maximum File Size: %{file_size} MB'
     no_required_files: There are no required files for this assignment.
     none: There are currently no assignments.
+    not_only_required_files: Students can submit other files in addition to required files
```

Comment:
You can actually remove this message---only show the `only_required_files` string when the option is enabled, otherwise we won't show any text.